.github/scripts/toolchain/build-mingw.sh

```diff
@@ -62,9 +62,11 @@ if [[ "$RUN_CONFIG" = 1 ]] || [[ ! -f "$MINGW_BUILD_PATH/Makefile" ]]; then
             --host=$TARGET \
             --enable-static \
             --enable-shared \
+            --with-libraries=all \
             $HOST_OPTIONS \
             $TARGET_OPTIONS \
-            CFLAGS="$CFLAGS"
+            CFLAGS="$CFLAGS" \
+            LDFLAGS="$LDFLAGS -L$TOOLCHAIN_PATH/lib/gcc/$TARGET/lib"
```

Comment:
For some unknown reason this is needed for `x86_64-w64-mingw32` build. Otherwise the linker cannot find `libgcc_s.la`.